### PR TITLE
add link website fahira

### LINKED ISSUE #23: FAQ

### PR DESCRIPTION
**Pull request by fahiraq**
I just add my link website (fahiraq) to the readme, please approve my pull request 

Fix #23

### CODE REVIEW

**Comment by fakhrip** [comment]
> I just add my link website to the readme, please approve my pull request xixi

You should put the `Fix #23` in the **description** of this PR, look at the [description of ferry's PR](https://github.com/Daskom-Lab/2021-Academy/pull/15#issue-716378809) for example...

Fix this and then i will continue reviewing your PR okay

**Comment by IritaSee** [comment]
Don't forget to add `mentors` as reviewer, because we will not be notified when you not assign us XD

File: README.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ Berikut merupakan orang-orang yang mengikuti academy ini :
   > Website link -> https://ferry5245.github.io/
 - [Fahira Qaulifa](https://github.com/fahiraq) [FAQ]
   > Sebagai mentee
+  > website link -> https://fahiraq.github.io/
```
Comment (fakhrip) [inline README.md]:
```suggestion
  > Sebagai mentee  
  > website link -> https://fahiraq.github.io/
```

Tambah 2 spasi juga setelah `Sebagai mentee` ya fahira, soalnya kalau ngga gitu ga akan ke enter di markdown, ini sama problem nya kayak dini dan yusril wkwk

DeepinScreenshot_select-area_20210822202104

**Review by fakhrip — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (IritaSee) [inline README.md:33]:
```suggestion
  >   website link -> https://fahiraq.github.io/
```
add some space so the markdown detect this as newline:

`
In a Markdown file or widget, enter two spaces before the line break, and then select Enter to begin a new paragraph. In a Markdown file or widget, enter two spaces before the line break, and then select Enter. This action begins a new paragraph
`

**Review by IritaSee — CHANGES_REQUESTED** [review]
Try to modify the website link so it is below the github profile

File: README.md (hunk 1 above)
Comment (IritaSee) [reply README.md:33]: oopss i think i marked the wrong line, follow @fakhrip instruction, then we can merge this, I will resolve this one

**Review by fakhrip — APPROVED** [review]

**Pull request merged**